src/pages/docs/guides/chat/build-livestream.mdx

```diff
@@ -0,0 +1,354 @@
+
+In making this decision, there are several considerations:
+
+* Should everyone see the same messages, or do you need to segment by language, region, friend group or topic?
+  * If you're running livestreams over multiple languages or geographical regions, you may want to consider separating users similarly so that everybody understands the chat. This can be achieved by using logged-in users provided location or using GeoIP to determine the location of each user.
+  * Alternatively, you may want to make use of translation services, that can translate messages on the fly to the language of the user. Many translation APIs including Google Translate are available and modern devices often come with on-device translation, which enables everyone to receive the same messages but in their own language.
+* Do you want an environment where everyone has time to read every message, or the chat to go wild with hype when something exciting happens?
+  * At anything more than 5-10 messages per second, a chat becomes unreadable for the audience. This may be ok in the moment when a goal is scored, for example, but may degrade the experience if it's for the entire game.
+  * **Sharding** is one approach to this problem - separating users into smaller groups so that the chat is more manageable. This can be done randomly, or using some predefined scheme, e.g. language, geographical region.
+  * You can also simply _drop_  a proportion of messages on the client to keep it readable. If you're not worried about everyone seeing everything, this is a simple approach.
+  * You could rate-limit how often users send messages. This controls the rate of messages, but can negatively impact the user experience if they're not able to respond when they want to.
+* Do you need to support both public and private chat?
+  * Some livestream experiences may have one big stream chat, but also enable individual users and groups to have private conversations.
+  * In this situation, Ably's [horizontal and vertical scalability](/docs/platform/architecture/platform-scalability) means you can have one big room for public chat and many smaller rooms for private conversations.
+
+Ably gives you the flexibility to design for your users, not your infrastructure.
+
+## Message throughput and rate management
+
+Livestream chat is defined by high message rates and unpredictable spikes. Ably [is engineered](/docs/platform/architecture/platform-scalability) to handle these loads without degradation.
+
+* **Proven at scale:** Ably delivers over 500 million messages per day for customers, with individual rooms supporting over 1 million concurrent users.
+* **Server-side batching:** Reduce costs and network overhead by grouping messages before delivery. For example, [batching messages](#server-side-batching) over a 500ms interval can reduce the number of outbound messages by up to 8x in high-traffic streams.
```

Comment:
How did you get to 8x here?

If you are publishing 100 messages a second to 100 people, that's 10k messages a second.
Batch of 500ms reduces that to 2 messages a second to 100 people, i.e. 200 messages a second, a 50x reduction. 
If users expected 1,000s of messages a second, and batch to 500ms, that's a 500x saving. 
You are way too conservative here IMHO>

Reply:
It was based on the example in the batching section, but I agree that it paints a far too conservative picture - will reword